README.md

```diff
@@ -26,7 +26,9 @@ A data package is an abstraction that encapsulates and automates data preparatio
 1. `pip install quilt`
 
 ## Linux
-1. `sudo apt-get install libhdf5-serial-dev`
+1. Install the OpenSSL headers:
+   * Ubuntu: `sudo apt-get install libssl-dev`
```

Comment:
What will it take to avoid this dependency? These are only needed if users are compiling, right? Can we remove this if everything installs from a wheel?